src/ImsStorage/ImsCache.php

```diff
@@ -52,6 +52,9 @@ public function cacheAccessToken($key, $accessToken)
     public function getAccessToken($key)
     {
         $this->loadCache();
+        if (!isset($this->cache[$key])) {
+            return false;
+        }
 
         return $this->cache[$key];
```

Comment:
While it's not explicitly typed (which is maybe something I should do), this method typically returns a string. Given that, I'd rather not mix return types (`string|bool`). Rather than returning false here, let's return null:

`return $this->cache[$key] ?? null;`